feature-extensions/apigee/README.md

```diff
@@ -0,0 +1,62 @@
+## Summary
+- **Make sure you have an Apigee account with at least two environments**
+- **This installer will use whatever AWS credentials you have configured by running 'aws configure'**
+- **Please make sure you are using the same AWS credentials you used to install your Jazz deployment**
+
+## USAGE
+usage: setup.py [-h] [--aws-region AWS_REGION]
```

Comment:
You can try enclosing your sh command using ``` for better formatting.

Reply:
Done